shoreditch.php

```diff
@@ -158,4 +158,8 @@ function shoreditch_civicrm_pageRun(&$page) {
   if ($pageName == 'CRM_Contact_Page_View_Summary') {
     CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.shoreditch', 'js/contact-summary.js');
   }
+
+  if ($pageName == 'CRM_Case_Page_DashBoard') {
+    CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.shoreditch', 'js/close-cases-filters.js');
```

Comment:
why do we need this script at all? Are we changing the default behaviour of the page?

Reply:
There is a requirement to keep the filter dropdown closed by default. So when user lands on the civicase dashboard all filter accordions are closed.

The idea way is to have `collapsed` class applied to accordion wrapper div. But doing that will need HTML changes and doing that will require core changes.

i.e. why I have implemented it through JS.